CHANGELOG/CHANGELOG-1.6.md

```diff
@@ -0,0 +1,114 @@
+- Upgrade packaged Whereabouts version to v0.5.1. ([#3511](https://github.com/antrea-io/antrea/pull/3511), [@antoninbas])
+- Upgrade to go-ipfix v0.5.12. ([#3352](https://github.com/antrea-io/antrea/pull/3352), [@yanjunz97])
+- Upgrade Kustomize from v3.8.8 to v4.4.1 to fix Cronjob patching bugs. ([#3402](https://github.com/antrea-io/antrea/pull/3402), [@yanjunz97])
+- Fail in Agent initialization if GRE tunnel type is used with IPv6. ([#3156](https://github.com/antrea-io/antrea/pull/3156), [@antoninbas])
+- Refactor the OpenFlow pipeline for future extensibility. ([#3058](https://github.com/antrea-io/antrea/pull/3058), [@hongliangl])
+- Validate IP ranges of IPPool for Antrea IPAM. ([#2995](https://github.com/antrea-io/antrea/pull/2995), [@ksamoray])
+- Validate protocol in the CRD schema of Antrea-native policies. ([#3342](https://github.com/antrea-io/antrea/pull/3342), [@KMAnju-2021])
+- Validate labels in the CRD schema of Antrea-native policies and ClusterGroup. ([#3331](https://github.com/antrea-io/antrea/pull/3331), [@GraysonWu])
+- Reduce permissions of Antrea ServiceAccounts. ([#3393](https://github.com/antrea-io/antrea/pull/3393), [@tnqn])
+- Remove --k8s-1.15 flag from hack/generate-manifest.sh. ([#3350](https://github.com/antrea-io/antrea/pull/3350), [@antoninbas])
+- Remove unnecessary CRDs and RBAC rules from Multi-cluster manifest. ([#3491](https://github.com/antrea-io/antrea/pull/3491), [@luolanzone])
+- Update label and image repo of antrea-mc-controller to be consistent with antrea-controller and antrea-agent. ([#3266](https://github.com/antrea-io/antrea/pull/3266) [#3466](https://github.com/antrea-io/antrea/pull/3466), [@luolanzone])
+- Add clusterID annotation to ServiceExport/Import resources. ([#3359](https://github.com/antrea-io/antrea/pull/3359), [@luolanzone])
+- Do not log error when Service for Endpoints is not found to avoid log spam. ([#3256](https://github.com/antrea-io/antrea/pull/3256), [@tnqn])
+- Ignore Services of type ExternalName for NodePortLocal feature. ([#3114](https://github.com/antrea-io/antrea/pull/3114), [@antoninbas])
+- Add powershell command replacement in the Antrea Windows documentation. ([#3264](https://github.com/antrea-io/antrea/pull/3264), [@GraysonWu])
+
+### Fixed
+
+- Add userspace ARP/NDP responders to fix Egress and ServiceExternalIP support for IPv6 clusters. ([#3318](https://github.com/antrea-io/antrea/pull/3318), [@hty690])
+- Fix incorrect results by `antctl get networkpolicy` when both Pod and Namespace are specified. ([#3499](https://github.com/antrea-io/antrea/pull/3499), [@Dyanngg])
+- Fix IP leak issue when AntreaIPAM is enabled. ([#3314](https://github.com/antrea-io/antrea/pull/3314), [@gran-vmv])
+- Fix error when dumping OVS flows of a NetworkPolicy via `antctl get ovsflows`. ([#3335](https://github.com/antrea-io/antrea/pull/3335), [@jainpulkit22])
```

Comment:
pedant nit: consider replacing "of a" with "for a"

Reply:
done